### ISSUE TITLE
Update Google Maps API Key

### ISSUE THREAD

**Issue by alex148** [rhiever/optimal-roadtrip-usa#8]
Hi, 

You should update your Google Maps API key, the full visualisation is broken (http://rhiever.github.io/optimal-roadtrip-usa/major-landmarks.html) since the recent Google update

:)

**Comment by rhiever** [comment]
Do you know if there is an API key that still works for free? All of my public maps have broken because Google Maps changed their API access quotas.

**Comment by iagomez** [comment]
I came here to open the same issue. Seeing your response has completely crushed my day

> All of my maps have broken because Google Maps changed their API access quotas.

Hope you can get it resolved and thank you for putting this together!

**Comment by individual8** [comment]
Actually there's $200 free monthly usage with Google Maps Platform.  

Find more info here: https://cloud.google.com/maps-platform/pricing/

**Comment by rhiever** [comment]
Yes, it is possible to use Google Maps for "free" (up to $200/month) if you enable billing, and that's how I still use my road trip optimization algorithm when creating maps locally. Others can do the same to generate new maps.

However, I am unwilling to attach my billing-enabled key to the public maps because I have no control over how much those maps are used, and I would rather not receive a surprise bill for thousands of dollars. :)

**Comment by individual8** [comment]
Unfortunately there isn't any other option. 😳

Maybe just advice people how to get their own API key and to get the demo running.

**Comment by Don-cloud** [comment]
have you guys tried using https://www.openstreetmap.org/ based on community based https://www.openstreetmap.org ?
